src/components/VmDetails/cards/DetailsCard/FieldRow.js

```diff
@@ -1,24 +1,34 @@
 import React from 'react'
 import PropTypes from 'prop-types'
 
-import { FieldLevelHelp } from 'patternfly-react'
 import { Row, Col } from '_/components/Grid'
 import style from './style.css'
+import InfoToolTip from '_/components/OverlayTooltip/InfoTooltip'
+import { tooltipPropType, tooltipPositionPropType } from '_/propTypeShapes'
 
-const FieldRow = ({ label, children, id, tooltip }) => (
+const FieldRow = ({ label, children, id, tooltip, tooltipPosition }) => (
   <Row className={style['field-row']}>
     <Col cols={5} className={style['col-label']}>
       <span>{label}</span>
-      {tooltip && <FieldLevelHelp buttonClass={style['field-level-help']} disabled={false} content={tooltip} inline />}
+      {tooltip &&
+        <span>
```

Comment:
Span required?

Reply:
The PF icon need to be wrap in `<div>` or `<span>` in some cases (like the `<FieldLevelHelp />` component)